src/main/webapp/WEB-INF/web.xml

```diff
@@ -22,6 +22,13 @@
         <param-name>org.jboss.weld.context.conversation.lazy</param-name>
         <param-value>false</param-value>
     </context-param>
+    <!-- UI warnings that can be enabled setting to Develop or another level. 
+            See https://docs.oracle.com/javaee/6/tutorial/doc/bnaxj.html#giqxl -->
+    <context-param>
+        <param-name>javax.faces.PROJECT_STAGE</param-name>
+<!--        <param-value>Development</param-value>-->
+        <param-value>Production</param-value>
+    </context-param>
```

Comment:
I'm very happy to see this in here because I always forget how to enable debugging in JSF.